lib/Ohmbrewer_Heating_Element.h

```diff
@@ -67,10 +69,39 @@ namespace Ohmbrewer {
              * Destructor
              */
             virtual ~HeatingElement();
-            
-            /**
-             * Overloaded << operator.
-             */
+
```

Comment:
No, I'm talking about the commented out pin stuff. Since you've put the pin stuff in Relay and TemperatureSensor, any references that are commented out (like the getPowerPin method below), should be cleaned up.